app/models/alchemy/essence_picture_view.rb

```diff
@@ -59,7 +61,9 @@ def img_tag
         essence.picture_url(options.except(*DEFAULT_OPTIONS.keys)), {
           alt: essence.alt_tag.presence,
           title: essence.title.presence,
-          class: caption ? nil : essence.css_class.presence
+          class: caption ? nil : essence.css_class.presence,
+          srcset: srcset.join(', ').presence,
+          sizes: options.fetch(:sizes, []).join(', ').presence
```

Comment:
The `fetch` here shouldn't be necessary (and isn't used for `srcset`), as `options` is initialized with `sizes` present, I think?